### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by mayank-aggarwal-yilu**

### CODE REVIEW

**Review by guneriu — DISMISSED** [review]
LGTM

File: yilu-common/values.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ readinessProbe:
 secrets:
   enabled: false
   name: ""
-  refreshInterval: 24h
+  refreshInterval: 360h
```
Comment (guneriu) [inline yilu-common/values.yaml:23]: How the scenario would be here? Imagine I've updated my secret in the Vault and now what should I do to get the secrets updated? 

in my opinion, this value should be something like 1h, if we see it creates a load on the Vault then we can adjust the Vault.  Let me know what you think.

Comment (mayank-aggarwal-yilu) [reply yilu-common/values.yaml:23]: The the forceful way to update the secret would be to just delete the k8s object, External Secrets Operator will recreate the secret. I will check in the config if there is some better way to update on demand.

Looking at the services we have I think even `1h` is fine, it shouldn't put a much load on the system. Though I was thinking more in terms of these being Static Secrets so will not be updated that often anyways.

**Review by guneriu — APPROVED** [review]
LGTM

**Pull request merged**